src/libraries/System.Text.Json/tests/Serialization/TestClasses/TestClasses.GenericCollectionsStruct.cs

```diff
@@ -0,0 +1,343 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We don't need a new file here. Can we move the contents to `TestClasses.GenericCollections.cs`. This applies to the other new file below.